client/styles/components/_editor.scss

```diff
@@ -96,6 +97,7 @@ pre.CodeMirror-line {
     right: 0;
     width: 100%;
     margin-left: 0;
+    z-index: 5;
   }
 
   z-index: 1;
```

Comment:
Let's change the existing `z-index: 1` (line 101) to `z-index: 5`, regardless of breakpoint.  Like it's not an issue on larger screens because the dialog doesn't overlap the gutter, but it does seem "right" that it should have a higher z-index.